test/page-objects/FormsRunner.page.js

```diff
@@ -1,4 +1,4 @@
-import { Page } from 'page-objects/page'
+import { Page } from './page.js'
```

Comment:
Now you've fixed these imports, can we remove the old aliases from the Babel config?

```patch
- "page-objects": "./test/page-objects",
- "components": "./test/components"
```

Then I presume we should stick to the `~` pattern until we discuss alternatives?

```suggestion
import { Page } from '~/test/page-objects/page.js'
```

Reply:
done